src/components/editor-content/editor-content.tsx

```diff
@@ -665,15 +690,267 @@ const EditorContent: React.FC<Props> = () => {
       </Accordion>
     );
   };
+
+  /**
+  * Test new vacation calculations
+  */
+  function getVacationDays(startDate: Date, endDate: Date, holidays: Date[]): number{
+    const totalDays = Math.round((endDate.getTime() - startDate.getTime()) / (1000 * 60 * 60 * 24))
+    let vacationDays = 0
+    let currentDate = new Date(startDate)
+    for (let i = 0; i <= totalDays; i++) {
+        if (currentDate.getDay() != 0 && !holidays.some(holiday => holiday.getTime() === currentDate.getTime())) {
+            vacationDays++
+        }
+        currentDate.setDate(currentDate.getDay() + 1)
+    }
+    return vacationDays
+  }
+
+  const startDate = new Date(selectedVacationStartDate)
+  const endDate = new Date(selectedVacationEndDate)
+
+  const holidays = [
+    new Date('2023-01-01'),
+    new Date('2023-01-06'),
+    new Date('2023-04-23'),
+    new Date('2023-04-25'),
+    new Date('2023-04-26')
+  ]
+
+  const vacationDays = getVacationDays(startDate, endDate, holidays)
+  
+  /**
+   * Renders and calculates days spend for vacation
```

Comment:
If I understand correctly spent is clearer than spend here.
```suggestion
   * Renders and calculates days spent for vacation
```